selfdrive/car/gm/values.py

```diff
@@ -98,6 +99,7 @@ def init_make(self, CP: car.CarParams):
   CAR.CADILLAC_ATS: GMCarInfo("Cadillac ATS Premium Performance 2018"),
   CAR.MALIBU: GMCarInfo("Chevrolet Malibu Premier 2017"),
   CAR.ACADIA: GMCarInfo("GMC Acadia 2018", video_link="https://www.youtube.com/watch?v=0ZN6DdsBUZo"),
+  CAR.BUICK_LACROSSE: GMCarInfo("Buick LaCrosse Premium 2017", "Adaptive Cruise Control (ACC) & LKAS"),
```

Comment:
Does it just support 2017, or should go all the way to 2019 like you put in CARS.md?

Reply:
If you have Adaptive Cruise Contro, LKAS can still be a separate package?

Reply:
It’s probably 2017-19. According to the window sticker LKAS was standard equipment for the Premium but ACC was an option.

Reply:
Changed to 2017-19 and removed LKAS description